src/utils.js

```diff
@@ -24,4 +24,8 @@ export function openGithubEvent(url) {
bAQFW4AixSJGzr0wGvvjYcTT81fbMhiLmskKIo1NfX91SJSy5h4sSJTJ06lerqak5w2Tc02U766kh41pLYLg4+44wzetoUXi8XX3wxdXV11NfXU1VV5dInISm1kp6j6qfSsGF4ZgcMrZ4CQrjl6J9t987PrFmzqKioAGD16tVs3LiRF154gdmzZ1NeXp6m/0bccd3hHXGgyUoPg5kzZ+LxJEpvNBpl9erVbN68mUOHDjFqVPqu9KgtU5ybRLYEQ4dP6a2fnNjX3/gTNP10N7oakJI5Xp1UJPD7/Zx//vmUl5fj9Xo5ePAgo0eP5pxzzkmb3AFLclXYSmZsvfBgghCMMZLXpLS0lMmTJ9PW1sb+/fvZvn07Q4YMYfbs2eh6elV6PypZEZXJW+HeYGg7B1n265fcecCefe9gGGencepOecVvMNHQ8iZCDwVM7oglAr3g2DEGfvIRwnEIDqmmddgITlFg3UCDIsUdb0zTRFVVFCUzf/vlEYulsQx0RwJm/F0mVH0rgwP+0oKuD3LfrkK9ovD7Ei/ePMjazpjDue1x5lhhzjqwh0cXLEaEOno4xuVzKLvoEsqLB/GTkvyc/GFMMvmInXFrjQQs8wgThpS7YUABqlaSlj+9kHq1LXk8GMfp58Q+iTtcHTB5ONTE7w6uZcyH65OMB3jv6We4rflFTjj8ES8F7X4bH7BhQWsKwOJii6KXAAXpGPDmpoEUl9yK6AVOLk2L12zJIFsyzlBRs0SCDbwRtvlOu0WL4/Afh9ejS5uBxQX4dUEsZlLs1TlzVCU/nT6BYYNLOCl2gBdkBZ8IP6d4lKzPAGizYcERh+et1H6Diy2gcsllv+WpPwSTU+CT5mqE+Mxtf+4WTj9UBdf6dcZ4lDRgtIAPYg6PBW2eMLv26TaB5pWk8j0pIZVibCisZXrRWGZocEOxylifQHV9NQB/CkvuOuawwaGHt2Rvpg7jtLKm5L2AIwWKyJw/KQM9a8Gz7SZnCsF0QzBcEdjAPhvWxB12Oan3qTSUnEl9+xZUnM5wVBFIcHpC/phawnO+EYBglQWrjjpcpMKlPkGNLjAUsBzYZ8KasORFy2VPl2PXKVlj55+H4itoymsgmWkC6Tr1doQbtA4mDC4HJbGuwoyz73ATa+N+lhvlNPWGJpkBzfOaZ1dvOFLN2UObk0Fw+e+COI6dRlJwa0OnE6XcdASrlQJa/ENANUCoIFSk4aO97ETmGxUJ4zOAcDq4ZW+Xp3WdHGnzxIPB9Crw2/vbkSLi1qnJarDMbHDP0bNJ2RpL4ETvhWmMyMzt8H61y0WfXSccIiy7t92dB+z69D0MY2wmptZn2Lue6xuUxqiCsxT4yIZ37f61y3MF67R5mfH3mTR4nHtDxJHvI8XY3HNdZGZcGZ2X0LlUVVhWU4TsdW3e/iD/HndyyOPjwCRbvpe5I2RaWzC47Isy+FqPyl5Lsq5rg9JLvALKPcnFrUCIDGOJPKIxQyRa1tbMDohG36ag6AtZYSRUaQqttpMo9p06t/g0qnSFUk3kFtLHa3CqRCNvZ3bAz368g0fXtKBq5VnzL5cc7QaqHp0rKryMKzH6Nj6vUpuNBAG23cKts3dk7gds22pj2msyozA5IHVKQwIXnWwtMrIgfrejUq7jUpl665nWGnZttfvuCgc7nsTjvfJ4CMgFquBbPpVRXpUSRXCSrrAglw1O50R/6lWo6JUiAsFTQZs9DtlDv695BgNPZm+Lr135FnOu/hjdGJ0v45rm17hpZGGSyoLtbbl9USPhmiovYwckNzt27+xgT1z2I/1S5mmae3hhxVvZm6L3L3CIxR9wJTm5Mi7Zxwr35YWk1MlwLdf0S51nPPYgD893cns1tvHNx7Hspn5RzM7rvy/VuaDMSKsdrwz1MUMTfWMAsLbKYHhh+t7v7pFe6jWRHyZZdhNb3ng893eDN/8oSjD4y/5QzHo98Xb2zFKd8QPSUf6CSg/DNSVr+I/0axTp6XonFmsMUAT3D9CY61PTo5EMEYuAcPCX3Danny9HX13zBLH4pswonHhQjRD81xAvz44pTufl/RUpsjqorkhl+bgC3q3xUG8Ism7eYrFNvN7wRKYhMzffFt3ocM53rqKsfBtC8bpN8IEynSuGeinr7uTmkOPkWAb7wBBVwORynWcH6rxxyOTDdhsBvBZwWGvKXrzDiRJou4p7r3P67wCAi07ezYbPbsVf9JsklPar3DTMS02hlr0PkG1TlcPL6kxiKDB1iM7UIYmKYX8QY22r3fOcUPBWLq3Z3dcY2b9FmDJsKdHI012h9vIwL0tPLkw3PhcHyCxG5nJ/rjqxyNNMG7I0m3m5fYyxc+uVxM1GJIwr0lBEX2XsOHI8WwTklEIC4mYj722+MhfTcnPAtd+LcKh5Oqa5M//wPt70yPF+y9zJ4abp3DQ98sU5AGDGaW00/fU8wzE3972C4vhXMB+iBJQKazMH/noec8a25WpW/75HmlV37IM/bTk/Eoms+tJyOM8IiUQiq04+tv18rhh/rD9D9vsd1JQpU8LA9wOBwDy/379IURQtZww43hx3ezfgOFYoFPpFcXHxPfm4N99fd8ji4uIlLS0tk03T3P3VVYHke03T3N3S0jK5uLh4Sb6xdVw/b6msrNy6YsWK2o6Ojlsc2z6Wzwr2L0ISh23bxzo6Om5ZsWJFbWVl5dbjeeJx/75n7ty55rRp0+7b+/FHNURCCzVpt/QNYvljhO7YLfuPhhbu2bOnZtq0affNnTvX5H+VnFTLq+vXeVtbW+dEo9H1tm3HZS/52bao3NViyUxyzeao3H0k+bpt2/FoNLq+tbV1zl1/fMnL6An8n5HGxsay1tbWmeFweGk0Gt1yzaZQcFeL6WR0wKaos+tQPBiNRreEw+GlbW1tMxsbG8u+zDl+1b96UnZ++HGlV5GV1dXVZQUFBRpAMBg0Dxw4cOzhD82DD1w69hD0+xOEb+QbyVP+B2mHaLGyYdwnAAAAAElFTkSuQmCC';
```

Comment:
It is tooooooo big. Maybe a small version of the logo in base64 can be a little smaller. It is too long for an URL icon.